tests/test_main.cpp

```diff
@@ -88,6 +89,11 @@ int test_main(int argc, char *argv[]) {
 	}
 	OS::get_singleton()->set_cmdline("", args);
 
+	TranslationServer *translation_server = nullptr;
+	translation_server = memnew(TranslationServer);
+	translation_server->setup(); //register translations, load them, etc.
+	translation_server->load_translations();
```

Comment:
Should be handled by #46313, will need to rebase later.

And sorry, I forgot that the test environment is setup in `Main::test_setup()`, not `test_main()`.

Reply:
I rebased this PR over #46313